charts/external-es-proxy/templates/external-es-proxy-deployment.yaml

```
@@ -87,6 +87,16 @@ spec:
               subPath: setenv.lua
             - name: tmp
               mountPath: /tmp
+            - name: nginx-client-body-temp
+              mountPath: /usr/local/openresty/nginx/client_body_temp
+            - name: nginx-proxy-temp
+              mountPath: /usr/local/openresty/nginx/proxy_temp
+            - name: nginx-fastcgi-temp
+              mountPath: /usr/local/openresty/nginx/fastcgi_temp
+            - name: nginx-uwsgi-temp
+              mountPath: /usr/local/openresty/nginx/uwsgi_temp
+            - name: nginx-scgi-temp
+              mountPath: /usr/local/openresty/nginx/scgi_temp
```

I saw @ianbuss's comment above that was resolved. I actually like the original approach of having a single volume, but agree with his concerns. The best solution for that is to use subPath, which will create a directory in the volume and mount that `volume/subPath` at the `mountPath`
```suggestion
            - name: tmp
              subPath: openresty/nginx/client_body_temp
              mountPath: /usr/local/openresty/nginx/client_body_temp
            - name: tmp
              subPath: openresty/nginx/proxy_temp
              mountPath: /usr/local/openresty/nginx/proxy_temp
            - name: tmp
              subPath: openresty/nginx/fastcgi_temp
              mountPath: /usr/local/openresty/nginx/fastcgi_temp
            - name: tmp
              subPath: openresty/nginx/uwsgi_temp
              mountPath: /usr/local/openresty/nginx/uwsgi_temp
            - name: tmp
              subPath: openresty/nginx/scgi_temp
              mountPath: /usr/local/openresty/nginx/scgi_temp
```
We do not have to do it this way, but it does create far fewer volumes, which seems cleaner.

---

Agree, subPath is The Way